webpack/makeconfig.js

```diff
@@ -69,7 +69,7 @@ module.exports = function(isDevelopment) {
         loaders: isDevelopment ? [
           'react-hot', 'babel-loader'
         ] : [
-          'babel-loader'
+          'babel-loader?optional[]=optimisation.react.inlineElements'
```

Comment:
"Starting with React 0.14..." Este are not there yet, it's dangerous I believe. I will remove it for now. As always, thank you for your effort.